html-forms/index.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>HTML Forms</title>
+  </head>
+  <body>
+    <h1>Coding Forms!</h1>
+    <form>
+      <p>Email</p>
+      <input type="text" name="email">
+    </form>
+    <form>
```

Comment:
You should only have one `form` element, which holds all the form control elements as children.